src/org/labkey/test/util/WorkbookHelper.java

```diff
@@ -75,6 +78,15 @@ public int getWorkbookIdFromUrl(URL url)
         }
     }
 
+    public void deleteWorkbooksFromDataRegion(DataRegionTable dr)
+    {
+        dr.clickHeaderButton("Delete");
+        assertTrue(_test.acceptAlert().contains("Are you sure you want to delete the selected rows?"));
```

Comment:
Needs to support deleting a single workbook. Should also use existing method to log actual alert.
```suggestion
        assertAlertContains("Are you sure you want to delete the selected row");
```

Reply:
@labkey-tchad sorry, i fixed this yesterday but apparently didnt push the commit.  we actually need the plural conditional based on the number of checked rows.